packages/forma-36-react-components/src/components/Asset/AssetIcon/AssetIcon.stories.tsx

```diff
@@ -1,19 +1,30 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { text, select } from '@storybook/addon-knobs';
+import type { Meta, Story } from '@storybook/react/types-6-0';
 
-import { types, AssetType } from '../Asset';
 import { AssetIcon } from './AssetIcon';
+import type { AssetIconProps } from './AssetIcon';
+import { types } from '../Asset';
+import notes from './README.mdx';
```

Comment:
@denkristoffer is that working for you?

Reply:
Actually it's not, I still see the old Storybook docs page. Maybe we can loop @gui-santos in here, any ideas why these docs aren't being displayed?

Reply:
@denkristoffer yeah, we talked about it with @gui-santos. I tried to fix it, but I failed for now. We decided that it's not the most important thing right now. There is a small hack, that Gui showed me, done in the `preview.js` in the storybook config, where we potentially could fix it (hack it more 😀).

Reply:
☝️ exactly

more on how the DocsPage work [here](https://storybook.js.org/docs/react/writing-docs/docs-page) <- we can change how it renders with a bit of work

and what we were doing with the `README.md` were just what storybook was recommending for the migration to v6 for people using addon-info/notes. Because addon-info/notes are binge deprecated and now it's addon-docs (comes in v6 with addon-essentials) that does that job

more on the deprecation of the mentioned addon here:
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#60-deprecations